schema/schema.go

```diff
@@ -183,10 +183,21 @@ func (ta *Table) FindColumn(name string) int {
 	return -1
 }
 
+// Get TableColumn by Index index
 func (ta *Table) GetPKColumn(index int) *TableColumn {
 	return &ta.Columns[ta.PKColumns[index]]
 }
 
+// Get Index by column index. Return nil if the column is not a index.
```

Comment:
> Get Index by column index

What if the column has multiple indexes?

Reply:
Yes, I think this function is to get primary index only. `PKColumns` must be `PRIMARY`. I Will update.